components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/ToolbarFeature.kt

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.toolbar
+
+import mozilla.components.definition.toolbar.ToolbarConcept
+import mozilla.components.session.SessionManager
+
+/**
+ * Feature implementation for connecting a toolbar implementation with the session module.
+ */
+class ToolbarFeature(
```

Comment:
We should probably introduce a common interface `Feature` with a `start` and `stop` method but we can do that later as then we also need a "parent" feature module to host that interface.

Reply:
Yeah, I was thinking that too (and I had that in my prototype). I wasn't sure if this was just a thing required by those two specifically or if this is a pattern that will be followed by all features more or less.

concept-feature? :)